components/layout/display_list/builder.rs

```diff
@@ -1066,12 +1088,20 @@ impl FragmentDisplayListBuilding for Fragment {
             None,
             style.logical_border_width().to_physical(style.writing_mode),
             self.border_padding.to_physical(style.writing_mode),
+            build_border_radius(&absolute_bounds, style.get_border()),
             index,
         );
 
+        let previous_clipping_and_scrolling = state.current_clipping_and_scrolling;
```

Comment:
Here we could use that helper too, ideally moving the inner block to another function or what not.

Reply:
Yes this pattern is used two times already in the file. This PR adds two more I think. The helper could handle those all.